config.json

```diff
@@ -355,6 +355,21 @@
       ]
     },
     {
+      "uuid": "363b8810-0bd1-c480-b23c-6b5f629975e7452b123",
+      "slug": "collatz-conjecture",
+      "core": false,
+      "unlocked_by": "leap",
+      "difficulty": 3,
+      "topics": [
+        "Control-flow (conditionals)",
+        "Control-flow (loops)",
+        "Recursion",
+        "Exception_handling",
+        "Integers",
+        "Mathematics"
+      ]
+    },
```

Comment:
This exercise is somewhat similar to "triangle", so I put it here and as a "triangle" it is unlocked with "leap".

Reply:
Looks good